src/pages/questions/ProductInformation.jsx

```diff
@@ -138,47 +170,41 @@ const ProductInformation = ({ question }) => {
       />
       {!hideImages && productData?.images && (
         <Grid container rowSpacing={1.5} spacing={1}>
-          {getImagesUrls(productData.images, question.barcode).map((src) => (
-            <Grid
-              item
-              key={src}
-              style={{ display: "inline-flex", alignItems: "flex-start" }}
-            >
-              <Zoom>
-                <img
-                  src={src}
-                  alt=""
-                  loading="lazy"
-                  style={{ maxWidth: 300, maxHeight: 300 }}
-                />
-              </Zoom>
-              {flagged[src[src.length - 5]] ? (
-                <Tooltip title="Unflag Image">
-                  <IconButton>
-                    <FlagIcon
-                      onClick={() =>
-                        deleteFlagImage(
-                          src,
-                          question.barcode,
-                          src[src.length - 5]
-                        )
-                      }
-                    />
-                  </IconButton>
-                </Tooltip>
-              ) : (
-                <Tooltip title="Flag Image">
-                  <IconButton>
-                    <OutlinedFlagIcon
-                      onClick={() =>
-                        flagImage(src, question.barcode, src[src.length - 5])
-                      }
-                    />
-                  </IconButton>
-                </Tooltip>
-              )}
-            </Grid>
-          ))}
+          {getImagesUrls(productData.images, question.barcode).map(
+            (src, imgid) => (
```

Comment:
You introduce `imgid` that is in fact the array index (so not sure it matches with the id). My proposal was to modify `getImagesUrls` such that you can write

```suggestion
            ({ src, imgid }) => (
```

Reply:
`imgid` is actually not used, if possible please remove it